File: static/load-contacts.php
<?php
    header("Content-Type: application/json; charset=UTF-8");
    $myfiler = fopen("{{ url_for('static', filename='contacts.json') }}", "r") or die("Unable to open file to read!");
    $obj = fread($myfiler,filesize("{{ url_for('static', filename='contacts.json') }}"));
    fclose($myfiler);
    echo $obj;
?>

File: static/load-contacts.py
<?py
    header("Content-Type: application/json; charset=UTF-8");
    file_name = "{{ urlfor('static', filename='contacts.json' }}";
    instructionsFile = open(file_name,"r");
    instructionsFile.close();
    if instructionsFile == '':
    	instructionsFile = 1
	else:
		pass
    print(instructionsFile);
?>